packages/react/src/QuestionnaireForm/QuestionnaireForm.tsx

```diff
@@ -120,6 +143,32 @@ function QuestionnaireFormItemArray(props: QuestionnaireFormItemArrayProps): JSX
     setResponseItems(newResponseItems);
     props.onChange(newResponseItems);
   }
+  const firstItem = props.items[0];
+  const firstItemValue = getExtension(firstItem, 'https://medplum.com/fhir/StructureDefinition/page-sequence');
+
+  if (firstItemValue && firstItemValue.valueString === 'page') {
```

Comment:
The logic here looks at the first item. If it's a page, this renders the whole questionnaire using a stepper. We drop items that are not in a page. 

This logic will produce surprising behavior for authors who might accidentally forget to set a group as a "page", or mix groups and questions at a top level. 

A more graceful approach would be to group all consecutive non-page items into their own pages. 

E.g
[q1, page1, page2, q3, q4, page 3]
woud yield the following pages: 
[q1]
[page1]
[page2]
[q3, q4]
[page3]